packages/synapse-react-client/src/components/CreateOrUpdateAccessRequirementWizard/CreateOrUpdateAccessRequirementWizard.tsx

```diff
@@ -96,7 +96,7 @@ export const CreateOrUpdateAccessRequirementWizard: React.FunctionComponent<
             accessRequirementId={accessRequirementId!}
             onSaveComplete={saveSuccessful => {
               if (saveSuccessful) {
-                onComplete()
+                onComplete(accessRequirementId)
```

Comment:
Pass back the AR ID that we are working with.  On the SWC side, this will redirect to a page that is focused on the single AR (not subject based), which will work for this new type of AR configuration.